prime-router/src/main/kotlin/azure/CheckFunction.kt

```diff
@@ -98,7 +105,10 @@ class CheckFunction : Logging {
                 }
             }
         } catch (t: Throwable) {
-            responseBody.add("${receiver.fullName}: ${t.localizedMessage}")
+            logger.info("Exception in health check: ${t.message}: ${t.cause?.message ?: "No root cause"}")
+            logger.info(t.stackTraceToString())
+            responseBody.add("${receiver.fullName}: ${t.localizedMessage}: ${t.cause?.message ?: "No root cause"}")
+            responseBody.add(t.stackTraceToString())
             responseBody.add("**** ${receiver.fullName}: FAILED")
```

Comment:
Left in the stack trace in the response body for now for testing, but I will remove once someone else has had a chance to run this.